nss-day-cohort-45/tabloidfullstack-giant-gibbons
# Gt create tag

## Changes Made:
1. Admin users can create new Tags
## How to Check
1. Log in as an administrator
2. Select `Tag Management` from the nav bar; click the `Create Tag` button at the bottom of the tag list
3. Provide a name for the tag in the form and click Save. You should be redirected back to the list of tags, and your new tag should be present
